web/src/components/speaker/SpeakerCard.vue

```diff
@@ -35,7 +35,9 @@
               >
                 {{ $t('findSpeaker.pronouns', [pronouns]) }}
               </div>
-              <p class="pa-0 ma-0">{{ speaker.get('speaker_bio') }}</p>
+              <p class="pa-0 ma-0">
+                {{ speaker.get('speaker_bio') }}
+              </p>
```

Comment:
sorry this was changed automatically when I ran `yarn lint --fix`. Maybe it got missed on some previous PR?

Reply:
Are you using prettier? We haven't been using it.

Reply:
It's possible that it passed some previous PR, cause we were not enforcing lint on CI at some point. Idk, at least I've had similar stuff on my PRs when running `yarn lint --fix` 🤔

Reply:
hmm I didn't change any settings, just `yarn install` and then before pushing I ran `yarn lint --fix`

Reply:
Okay, let's just take the fix and assume this is what eslint wants 😄